besu/src/main/java/org/hyperledger/besu/RunnerBuilder.java

```diff
@@ -1176,4 +1176,9 @@ public RunnerBuilder minPeers(final int minPeers) {
     this.minPeers = minPeers;
     return this;
   }
+
+  public RunnerBuilder legacyForkId(final boolean legacyEth64ForkIdEnabled) {
```

Comment:
can you explain legacy fork Id?

Reply:
Setting this changes the behaviour to always sending the last ForkId based on the genesis file and not the ForkId of where we up to at the moment.

Reply:
for what is it used?